Command/CKEditorInstallerCommand.php

```diff
@@ -256,50 +259,50 @@ private function title(OutputInterface $output)
      */
     private function comment($message, OutputInterface $output)
     {
-        $output->writeln(' // '.$message);
+        $output->writeln(' // ' . $message);
         $output->writeln('');
     }
 
     /**
-     * @param string          $message
+     * @param string $message
      * @param OutputInterface $output
      */
     private function success($message, OutputInterface $output)
     {
-        $this->block('[OK] - '.$message, $output, 'green', 'black');
+        $this->block('[OK] - ' . $message, $output, 'green', 'black');
     }
 
     /**
-     * @param string          $message
+     * @param string $message
      * @param OutputInterface $output
      */
     private function info($message, OutputInterface $output)
     {
-        $this->block('[INFO] - '.$message, $output, 'yellow', 'black');
+        $this->block('[INFO] - ' . $message, $output, 'yellow', 'black');
     }
 
     /**
-     * @param string          $message
+     * @param string $message
      * @param OutputInterface $output
-     * @param string          $background
-     * @param string          $font
+     * @param string $background
+     * @param string $font
      */
     private function block($message, OutputInterface $output, $background = null, $font = null)
     {
         $options = [];
 
         if ($background !== null) {
-            $options[] = 'bg='.$background;
+            $options[] = 'bg=' . $background;
         }
 
         if ($font !== null) {
-            $options[] = 'fg='.$font;
+            $options[] = 'fg=' . $font;
         }
 
         $pattern = ' %s ';
 
         if (!empty($options)) {
-            $pattern = '<'.implode(';', $options).'>'.$pattern.'</>';
+            $pattern = '<' . implode(';', $options) . '>' . $pattern . '</>';
```

Comment:
I would refrain from CS fixes and save them for another PR. Makes reviewing harder.

Reply:
problem is: my IDE did all the changes in files i visited, to remove them will be very hard :-(